presto-hive/src/main/java/io/prestosql/plugin/hive/metastore/thrift/ThriftHiveMetastore.java

```diff
@@ -1310,6 +1321,12 @@ private boolean containsAllPrivilege(Set<PrivilegeGrantInfo> requestedPrivileges
     private RetryDriver retry()
     {
         return RetryDriver.retry()
+                .exponentialBackoff(
```

Comment:
Nit: this is short enough that it would look better on a single line
```java
.exponentialBackoff(minBackoffDelay, maxBackoffDelay, maxRetryTime, scaleFactor)

Reply:
ok